### PR TITLE
[DC-3280] Add a device_id deidentification check to `cdr_deid_qa_report6.py`

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py [hunk 1]
```diff
@@ -532,6 +533,74 @@
...
+FROM check_uuid_unique
+  
+""").render(project_id=project_id,
+            pipeline=pipeline,
+            deid_cdr_fitbit=deid_cdr_fitbit)
+
+df2 = execute(client, query)
+
+if sum(df2['bad_rows']) == 0:    
+    df = df.append(
```
Comment (MichaelSchmidt82) [inline data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py:590]: While not in scope, I wish we'd rename variable `df` to `summary`.

**Review by MichaelSchmidt82 — COMMENTED** [review]
@nishanthpp93 , would you mind double-checking the query?

**Review by MichaelSchmidt82 — APPROVED** [review]
looks good

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py [hunk 2]
```diff
@@ -534,6 +534,70 @@
...
+# * check_uuid_unique - All deidentified device_ids for a device_id/person_id should be unique. If not, replace the non-unique research_device_ids in the masking table with new UUIDs. 
+#
+# Device_id is being deidentified by a combination of the following cleaning rules.
+# 1. generate_research_device_ids - Keeps the masking table(wearables_device_id_masking) updated.
+# 2. deid_fitbit_device_id - Replaces the device_id with the deidentified device_id(research_device_id).
+#
+
+# +
+query = JINJA_ENV.from_string("""
+WITH not_research_ids AS (
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py:554]: Could be ambiguous
```suggestion
WITH not_research_device_ids AS (
```

**Review by nishanthpp93 — COMMENTED** [review]
Small change, but otherwise looks good!

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py (hunk 1 above)
Comment (brendagutman) [reply data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py]: I updated this query to be consistent with your changes in DC3170 that will soon be merged into develop. Using `summary` and `results`.

**Pull request merged**